### DISCUSSION TITLE
Suggestion: User custom specifying magnification of the detailed view

### DISCUSSION THREAD

**Discussion by Yangeok** [dineug/erd-editor#282] [Ideas]
Thank you so much for making a good product. While using AQueryTools, it was paid, so I was looking for a free product and using it.

- AS-IS: Table detail can only be viewed when it's over 80% magnification
- TO-BE:
  - It can be seen even at smaller magnification (e.g., 60% etc.)
  - User can adjust it in settings

Please let me know if the function that can be added or if add it, there will be no performance issues.

**Comment by dineug** [comment]
Please enter 71%.
Less than 70% is not seen properly, so I don't know if it's meaningful.
There is no performance issue.
